packages/dashboard/src/components/pages/about-us/about-us.tsx

```diff
@@ -24,7 +24,7 @@ const AboutUs: React.FC = () => {
             </Text>
             <div className='dw-about-us__wrapper'>
                 {Array.from(new Array(total_previews).keys()).map(key => (
-                    <Frame src={`${config_store.asset_path}/images/p${key + 1}.jpeg`} key={key} alt={`p${key + 1}`} />
+                    <Frame src={`${config_store.asset_path}/images/placeholder.png`} key={key} alt={`p${key + 1}`} />
```

Comment:
This may raise merge conflict as we discussed yesterday. I have removed all functionalities related to Frames.

Reply:
its just frame component that might be changed, but the usage like about us, can be changed/removed since we won't ever have the page in real implementation later on